packages/functional-tests/playwright.config.ts

```diff
@@ -63,13 +63,12 @@ export default defineConfig<PlaywrightTestConfig<TestOptions, WorkerOptions>>({
   ],
   reporter: CI
     ? [
-        ['./lib/ci-reporter.ts'],
         [
-          'junit',
+          'blob',
           {
             outputFile: path.resolve(
               __dirname,
-              '../../artifacts/tests/test-results.xml'
```

Comment:
I know this is draft still, but I'd probably recommend keeping JUnit and _adding_ the new blob reporter as well. If we remove the junit reporter here I'm guessing this will break Kat's tooling (which we want to keep until we can sort all this metrics pipeline stuff out). I'm not 100% certain what else downstream we're using the custom ci-reporter for, or what it might affect elsewhere in the universe.

Next we'll need to figure out how to copy the test reporters from each runner/shard into a single artifact location and then we can probably run [`merge-reports` tool](https://playwright.dev/docs/test-sharding#merge-reports-cli) to generate a unified juint file and unified HTML report, and anything else we want.

I found https://support.circleci.com/hc/en-us/articles/360043804531-Collecting-combining-files-after-parallel-test-runs, but to call that helpful is being generous. Not finding a lot of great results in the [CircleCI forums](https://discuss.circleci.com/t/merge-artifact-result/38598) on merging+storing the artifacts from multiple parallel runs, but maybe I'm searching bad terms. Or maybe Google Search is better than CircleCI search.

Reply:
Odd, I'm not seeing the blob ZIP files in the artifacts dir.
But that `outputFile` config should still be valid, per https://playwright.dev/docs/test-reporters#blob-reporter.